packages/data-provider/src/core/provider.ts

```diff
@@ -49,9 +66,16 @@ const provider = (configData: AppConfig) => {
 			submittedData: submittedDataService(baseDeps),
 		},
 		utils: {
+			audit: auditUtils,
+			convertSqonToQuery: convertSqonToQueryUtils,
+			dictionarySchemaRelations: dictionarySchemaRelationUtils,
 			dictionary: dictionaryUtils,
+			errors: errorUtils,
+			file: fileUtils,
+			schema: schemaUtils,
 			submission: submissionUtils,
 			submittedData: submittedDataUtils,
+			type: typeUtils,
 		},
```

Comment:
Since we're already exporting controllers and services, it makes sense to also export some utility functions as well especially for validations and other common operations.